src/Appium.Net/resources/scripts/path_to_default_node.sh

```diff
@@ -0,0 +1,4 @@
+#!bin/sh
```

Comment:
why this script cannot be executed directly from the code?

Reply:
something like https://loune.net/2017/06/running-shell-bash-commands-in-net-core/ with `UseShellExecute` set to true

Reply:
It can. Towards the end, I noticed we do this for Windows

Reply:
But there is an issue within mono failing to find bash when you try to start a process passing `/bin/bash/` as an application/file argument. Nothing is simple

Reply:
you don't need to use `/bin/bash` as executable if UseShellExecute is set to true

Reply:
You need UseShellExecute set to false in order to redirect the output stream

Reply:
I realised the cause of the win32 exception when looking for bash is down to a parsing issue of the arguments. Will update this PR to remove the embedded resource